mail_gateway/models/mail_channel.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2024 Dixmit
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import base64
+
+from odoo import api, fields, models
+
+
+class MailChannel(models.Model):
+    _inherit = "mail.channel"
+
+    token = fields.Char()
```

Comment:
This field name is too generic and it can collide with other. Can you change it?